### DISCUSSION TITLE
Adding a document with default data

### DISCUSSION THREAD

**Discussion by tbenbow** [tinacms/tinacms#2230] [Help] [answered]
How should I define default data when creating a new document with the "Add Document" modal?

I tried using the defaultItem property in my template but my new markdown document for my post only has a single property "_template: post" - it doesn't add the defaultItem data I define.

Here is my schema...
```
export default defineSchema({
  collections: [
    {
      label: "Posts",
      name: "posts",
      path: "content/posts",
      templates: [ post ]
    },
  ],
});
```

And here is the post template I am importing...
```
export const post: TinaTemplate = {
    name: "post",
    label: "Post",
    ui: {
      defaultItem: {
        title: "The Title",
      },
    },
    fields: [
      {
        type: "string",
        label: "Title",
        name: "title", 
      }
    ]
}
```

**Comment by jeffsee55** [comment, marked as answer]
@tbenbow `defaultItem` doesn't work at the collection level, it only works for `object`-like fields. This is something we should support though! I'll make this a ticket 👍
